common/src/main/java/com/tencent/rss/common/config/ConfigUtils.java

```diff
@@ -95,7 +97,20 @@ static Long convertToLong(Object o) {
     } else if (o.getClass() == Integer.class) {
       return ((Integer) o).longValue();
     }
-    return Long.parseLong(o.toString());
```

Comment:
convertToInt also need UnitConverter.

Reply:
Int type is used for other purposes (e.g., core numbers, replicas), rather size.

Reply:
int type may also be used for size.

Reply:
Using int type for size is unsafe.
So, we should use long type for all size types

Reply:
> 

why unsafe? int type may be enough for some size.

Reply:
The max value of int is 4GB, so Linux kernel uses long for all size types (i.e., size_t).